DB/src/main/java/io/deephaven/db/v2/utils/TableLoggers.java

```diff
@@ -75,4 +74,12 @@ public static QueryTable processInfoLog() {
     public static QueryTable processMemoryLog() {
         return ProcessMemoryTracker.getInstance().getQueryTable();
     }
+
+    /**
+     * Start collecting data for query update performance.
+     */
+    @ScriptApi
+    public static void startUpdatePerformanceLog() {
```

Comment:
You are right, the QPL and QOPL are effectively started automatically the first time they have data.
I think we should start the UPT from `LiveTableMonitor.start()`; if it becomes a memory problem, we will make it configurable later.